File: fizzbuzz/fizzbuzz.py
def fizzbuzz(n):
    # Implementation with off-by-one error
    for i in range(n + 1):  # Off-by-one error: includes n instead of stopping at n-1
        if i == 0:  # Another aspect of the off-by-one issue
            continue
        if i % 3 == 0 and i % 5 == 0:
            print("FizzBuzz")
        elif i % 3 == 0:
            print("Fizz")
        elif i % 5 == 0:
            print("Buzz")
        else:
            print(i)

if __name__ == "__main__":
    fizzbuzz(15)

File: fizzbuzz/fizzbuzz.js
function fizzbuzz(n) {
    // Implementation with string concatenation bug
    for (let i = 1; i <= n; i++) {
        let output = "";
        // Bug: String concatenation instead of exclusive conditions
        if (i % 3 === 0) output += "Fizz";
        if (i % 5 === 0) output += "Buzz";
        // This leads to "FizzBuzz" being printed as "BuzzFizz" in some cases
        if (output === "") output = i.toString();
        console.log(output);
    }
}

fizzbuzz(15);